### ISSUE TITLE
Feature Request: command line option to open in new tab

### ISSUE THREAD

**Issue by hewigovens** [git-up/GitUp#685]
It would be nice if we can run `gitup --new-tab` from cli

**Comment by ppo** [comment]
And `gitup --active-tab` to add it in the currently active window.

**Comment by lucasderraugh** [comment]
@ppo @hewigovens How would `--active-tab` be different than `--new-tab`? Wouldn't the behavior to take the existing main/key window and then adding a second tab to it? I'm thinking about adding this to GitUp `1.2.1`.

**Comment by hewigovens** [comment]
Looks same for me

**Comment by ppo** [comment]
@lucasderraugh @hewigovens Looks the same to me too! 😂

Sorry, I've no idea what I tried to explain! 🤷‍♂️ I was surely very tired and doing multiple things at the same time 😕

**Comment by lucasderraugh** [comment]
@hewigovens @ppo Addressing this in PR #734

**Comment by szhu** [comment]
I suspect that some of you actually want to always open repos in GitUp as tabs instead of windows, even from the GUI.

You can configure that this way:

```
defaults write co.gitup.mac AppleWindowTabbingMode always
```

To undo:

```
defaults delete co.gitup.mac AppleWindowTabbingMode
```

---

More notes about this setting:

- This trick works with ANY app that uses native macOS tabs; just change the app id.
- If you want ALL apps that support native tabs to always open documents as tabs: `defaults write -g AppleWindowTabbingMode always`. Or you can find the option in System Preferences > General, which changes the same setting.
- The allowed values are `manual`, `fullscreen`, and `always`.

**Comment by lucasderraugh** [comment]
Closed with #734.
